File: JANUARY/26-01-2026/1200.cs
public class Solution {
    public IList<IList<int>> MinimumAbsDifference(int[] arr) {
        Array.Sort(arr);
        int mn = int.MaxValue;
        for (int i = 1; i < arr.Length; i++)
            mn = Math.Min(mn, arr[i] - arr[i - 1]);

        IList<IList<int>> res = new List<IList<int>>();
        for (int i = 1; i < arr.Length; i++)
            if (arr[i] - arr[i - 1] == mn)
                res.Add(new List<int> { arr[i - 1], arr[i] });
        return res;
    }
}

File: JANUARY/26-01-2026/1200.ts
function minimumAbsDifference(arr: number[]): number[][] {
    arr.sort((a, b) => a - b);
    let mn = Infinity;
    for (let i = 1; i < arr.length; i++)
        mn = Math.min(mn, arr[i] - arr[i - 1]);

    let res: number[][] = [];
    for (let i = 1; i < arr.length; i++)
        if (arr[i] - arr[i - 1] === mn)
            res.push([arr[i - 1], arr[i]]);
    return res;
}